numba/tests/test_array_manipulation.py

```diff
@@ -174,6 +174,9 @@ def numpy_fill_diagonal(arr, val, wrap=False):
 def numpy_shape(arr):
     return np.shape(arr)
 
+def numpy_size(arr):
```

Comment:
This function also needs a corresponding runner function. See: https://github.com/numba/numba/blob/108560408eb3456c53bb08426eec1ea62f95ff12/numba/tests/test_array_manipulation.py#L1198

Reply:
> This function also needs a corresponding runner function. See:
> 
> https://github.com/numba/numba/blob/108560408eb3456c53bb08426eec1ea62f95ff12/numba/tests/test_array_manipulation.py#L1216

If I implement this loop in test_size() it fails for some reason

Reply:
> This function also needs a corresponding runner function. See:
> 
> https://github.com/numba/numba/blob/108560408eb3456c53bb08426eec1ea62f95ff12/numba/tests/test_array_manipulation.py#L1216

I can't seem to replicate this loop pattern in test_size() without the tests failing,any tips?